client/asset/eth/nodeclient.go

```diff
@@ -209,6 +209,8 @@ func (n *nodeClient) balance(ctx context.Context) (*Balance, error) {
 
 	for _, tx := range pendingTxs {
 		from, _ := ethSigner.Sender(tx) // zero Address on error
+		// Intentionally ignoring receives from addresses that do not
```

Comment:
I don't think those would even be listed in pending transactions... I think this is more of a sanity check.

Reply:
Is "those" receives from addresses that do not belong to the wallet? I think this will show all txs in the mempool?

Reply:
From **go-ethereum/light/txpool.go**

```Go
// TxPool implements the transaction pool for light clients, which keeps track
// of the status of locally created transactions, ...
```

More accurately, `ignoring transactions that are not from us`, but that's really what the code says, so I don't know that the doc adds much.

Reply:
Removing the doc.